src/bp/ml/svm/addon/index.ts

```diff
@@ -14,19 +14,16 @@ export default class BaseSVM {
   }
 
   static restore = (model: Model) => {
-    const random_seed = parseInt(process.env.NLU_SEED || '')
-    const clf = random_seed ? new addon.NSVM({ random_seed }) : new addon.NSVM()
-
+    const clf = new addon.NSVM()
```

Comment:
if a model was previously trained with a seed, we don't need to pass this seed to the constructor ?

Reply:
nope, the seed is only used at training time to build the model. For a given model, results are always the same.